SAP/luigi
# Receive more messages from mfe

**Description**

Changes proposed in this pull request:

- Add more message types
- Add examples to test
- 

**How to test/try out:**
- cd container
- npm run start
- Go to http://localhost:8080/index2.html 
- Click on the buttons on the shown microfrontend to trigger the client calls and check that the messages are received properly
